src/transport_picoquic.cpp

```diff
@@ -1150,13 +1153,7 @@ void PicoQuicTransport::on_recv_stream_bytes(StreamContext* stream_cnx, uint8_t*
 
     bool object_complete = false;
 
-    if (stream_cnx->stream_rx_object != nullptr && stream_cnx->stream_rx_object_size == 0) {
-        logger->warning <<  "on_recv_stream_bytes has object size zero and non-null rx_object sid: "
-                        << std::to_string(stream_cnx->stream_id) << std::flush;
-
-    }
-
-    if (stream_cnx->stream_rx_object == nullptr || stream_cnx->stream_rx_object_size == 0) {
+    if (stream_cnx->stream_rx_object == nullptr) {
```

Comment:
Only nullptr should be allowed.  Delete later in this block would error out if null but object size is zero.